RUFAS/end_to_end_tester.py

```diff
@@ -0,0 +1,91 @@
+from collections import namedtuple
+from deepdiff import DeepDiff
+from pathlib import Path
+from typing import Any
+
+from RUFAS.output_manager import OutputManager
+from RUFAS.units import MeasurementUnits
+
+import json
+import re
+
+
+TestResultPaths = namedtuple("TestResultPaths", ["domain_prefix", "expected_results_path", "actual_results_path"])
```

Comment:
```suggestion
TestResultPathType = namedtuple("TestResultPaths", ["domain_prefix", "expected_results_path", "actual_results_path"])
```
You are defining the type here; the name implies that a collection or iterable is being defined.